src/pages/zh-cn/core-concepts/astro-components.md

```diff
@@ -202,33 +259,43 @@ const { greeting, name } = Astro.props
 
 你还可以使用 TypeScript 导出 `Props` 类型接口来定义参数。Astro 将自动选择任何导出的 `props` 接口，并为你的项目提供类型警告/错误提示。当从 `Astro.props` 解构时，这些参数也可以被赋予默认值。
 
-```astro
+```astro /(\w+)=\S+/
+---
+// src/components/GreetingCard.astro
+import GreetingHeadline from './GreetingHeadline.astro';
+const name = "Astro"
 ---
-//src/components/GreetingHeadline.astro
+<h1>Greeting Card</h1>
+<GreetingHeadline greeting="Hi" name={name} />
+<p>I hope you have a wonderful day!</p>
+```
+
+你也可以通过导出 `Props` 类型接口，用 TypeScript 定义来参数。Astro 会自动接收任何导出的 `Props` 接口，并为你的项目提供类型警告/错误。这些道具也可以在从 `Astro.props` 解构时给出默认值。
+
+```astro ins={3-6}
+---
+// src/components/GreetingHeadline.astro
 export interface Props {
   name: string;
   greeting?: string;
 }
 
-const { greeting = "Hello", name } = Astro.props as Props;
+const { greeting = "Hello", name } = Astro.props;
```

Comment:
Up to you, but you could translate the prop values and text in these examples (and below) if you wanted.